jsapi/examples/simple.js

```diff
@@ -10,5 +10,7 @@ var map = new lux.Map({
   bgLayer          : 655,
   layers           : [ 'arrets_bus', 650 ],
   mousePosition    : true,
-  mousePositionSrs : 4326
+  mousePositionSrs : 4326,
+  bgSelector       : true,
+  bgSelectorTarget : 'mapContainer'
```

Comment:
We just decided that controls should be configured with the following syntax:
```
bgSelector: {
  target: element
}
```

All the controls must be configured with a target.